nf_core/__main__.py

```diff
@@ -2181,14 +2181,14 @@ def command_lint(
     help="The name of the GitHub organisation where the pipeline will be hosted (default: nf-core)",
 )
 @click.pass_context
-def command_create(ctx, name, description, author, version, force, outdir, template_yaml, plain, organisation):
+def command_create(ctx, name, description, author, version, force, outdir, template_yaml, organisation):
```

Comment:
We shouldn't remove the argument from this function, this makes the command to fail, you can test it by running `nf-core create`

Reply:
Also saw that we should give a default to the version, as we do with `nf-core pipelines create`